pkg/asset/installconfig/baremetal/baremetal.go

```diff
@@ -41,6 +41,34 @@ func Platform() (*baremetal.Platform, error) {
 		return nil, err
 	}
 
+	err = survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "External bridge",
+				Help:    "External bridge is used for external communication.",
+				Default: baremetaldefaults.ExternalBridge,
+			},
+			Validate: survey.ComposeValidators(survey.Required, uriValidator),
```

Comment:
The URI validator doesn't seem appropriate for the name of a bridge.

We could leave it unvalidated, or we could try to validate that a bridge with that name exists.  If you do that, it's probably not much more work to populate the list of bridges as choices for the interactive text UI.

If you want to leave it unvalidated for this PR, how about open a bug for adding validation later.

Reply:
Good catch, you caught me copy and pasting. I've added a validation that checks if the interface exists. It cannot tell us if it's a bridge, without making a more lower level system call but this is at least something.